R/bag_tree-rpart.R

```diff
@@ -1,15 +1,21 @@
 #' A wrapper for survival times with `survbagg` models
-#' @param object A model from `ipred::bagging()`.
+#' @param object A parsnip `model_fit` object resulting from `bag_tree()` with `engine = "rpart"`.
```

Comment:
I really appreciate this phrasing—if users are confused by the message and look to the docs, this should clear up confusion. That said, do we want [parsnip::bag_tree()] or [`bag_tree()` with `engine = "rpart"`][parsnip::details_bag_tree_rpart] here?

Reply:
That's a good call, thanks! I'll use the second option because the engine really matters here 👍